src/include/security/widgets.rb

```diff
@@ -261,22 +262,36 @@ def initialize_security_widgets(include_target)
             "Value"  => "100"
           }
         }
+
+      @WIDGETS.merge!(
+        @display_manager.shutdown_var_name => shutdown_login_manager_widget
+      ) if @display_manager
     end
 
-    def boot_option_labels
+    def shutdown_login_manager_widget
       {
-        "ignore" => _("Ignore"),
-        "reboot" => _("Reboot"),
-        "halt"   => _("Halt")
+        "Widget"  => "ComboBox",
+        # ComboBox label
+        # TRANSLATORS: %s will be the configured display manager usually: GDM or KDM,
+        # but could be XDM,WDM,ENTRANCE,CONSOLE
+        "Label"   => _(
+          "&Shutdown Behaviour of %s Login Manager:"
+        ) % @display_manager.name,
+        "Options" => shutdown_options,
+        "Value"   => @display_manager.shutdown_default_value
       }
     end
 
     def console_shutdown_options
       ::Security::CtrlAltDelConfig.options.map do |opt|
-        [opt, boot_option_labels[opt]]
+        [opt, BOOT_OPTION_LABELS[opt.downcase]]
```

Comment:
Constants are not translated, you need to translate them explicitly here: `_(BOOT_OPTION_LABELS[opt.downcase])`.

Note: `N_("foo")` does not translate (returns the original), it just marks the text to be found by gettext when creating the pot file. You need to explicitly translate by `_()` when the text is used.